documentation/website/docs/installation.md

```diff
@@ -70,6 +71,21 @@ $ source ~/.bashrc
 ```
 You should be able to open a new shell and run `pyre -h` now, confirming `pyre` was set-up correctly. Any changes made to the Pyre Python client code should be immediately observable the next time you invoke `pyre`
 
+#### Testing changes for Plugin Development
+VSCode often has problems working with aliases. As a result, Pyre fails to start in the VSCode Extension Development Host during the debugging process. To get around this, instead of creating an alias, we can create an executable script called `pyre` in the `scripts` directory with the following lines
+
+```bash
+#!/bin/bash
+python -m client.pyre "$@"
+```
+Add the `pyre-check` directory to `PYTHONPATH`, the `pyre-check/scripts` directory to `PATH` and then use the command `pyre` to launch the client like before
+
+```bash
+$ echo "export PYTHONPATH=/path/to/pyre-check:$PYTHONPATH" >> ~/.bashrc
+$ echo "export PATH=/path/to/pyre-check/scripts:$PATH" >> ~/.bashrc
```

Comment:
You should add quotes like we did for the alias example above, so that spaces in `PATH` doesn't break things